docs/_includes/body-end.html

```diff
@@ -25,6 +25,7 @@
 <script src="js/jquery.matchHeight-min.js"></script>
 <script src="js/index.js"></script>
 <script src="js/docs.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/prismjs@1.17.1/prism.min.js"></script>
```

Comment:
I think cloudflare is pretty popular, haven't heard of jsdelivr before: https://cdnjs.com/libraries/prism. Maybe use it from there? Also, dockerfile is used a lot in our code blocks, the prism.min.css only contains the core languages, but doesn't include add-on languages. For dockerfile, also add: https://cdnjs.cloudflare.com/ajax/libs/prism/1.17.1/components/prism-docker.min.js